core/src/main/scala/org/apache/spark/sql/rapids/tool/util/RapidsToolsConfUtil.scala

```diff
@@ -18,62 +18,53 @@ package org.apache.spark.sql.rapids.tool.util
 
 import org.apache.hadoop.conf.Configuration
 
-import org.apache.spark.SparkConf
-import org.apache.spark.deploy.SparkHadoopUtil
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.SparkSession
 
 /**
  * Contains util methods to interact with Spark/Hadoop configurations.
  */
-
 object RapidsToolsConfUtil extends Logging {
-  val hadoopConfPrefix = s"${RAPIDS_TOOLS_SYS_PROP_PREFIX}hadoop."
-  val sparkConfPrefix = s"${RAPIDS_TOOLS_SYS_PROP_PREFIX}spark."
-
-  private lazy val configMap = loadConfFromSystemProperties
+  private val HADOOP_CONF_PREFIX = s"${RAPIDS_TOOLS_SYS_PROP_PREFIX}hadoop."
```

Comment:
I would make this more specific, RAPDS_TOOLS_HADOOP_CONF_PREFIX or something. hadoop conf prefix makes it sounds like its hadoop.xxxx

Reply:
the only other thing I am thinking about is whether we just use the spark.hadoop.XXXX prefix.  we could easily just look at that too and wondering if it would make users life easier if they already have those specified in their spark applications.